### PR TITLE
[TE] rootcause poc - basic metric tabs with sorting

### PR DESCRIPTION
**Pull request by apucher**
* RCA page tabs for metrics, events, dimensions

* metric tab now displays boxes with change and is sortable

* heatmap refreshes when metric is unselected

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js [hunk 1]
```diff
@@ -1,6 +1,13 @@
 import Ember from 'ember';
 import { toBaselineUrn, hasPrefix, filterPrefix } from '../../../helpers/utils';
 
+const ROOTCAUSE_METRICS_SORT_PROPERTY_METRIC = 'metric';
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js]: I feel that having this in one const PROP will be more readable: 
```
const ROOTCAUSE_METRICS_SORT_PROPERTY = {
  DATASET= 'dataset',
  CHANGE= 'change',
  METRIC='metric'
}
```

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+const ROOTCAUSE_METRICS_SORT_PROPERTY_DATASET = 'dataset';
+const ROOTCAUSE_METRICS_SORT_PROPERTY_CHANGE = 'change';
+
+const ROOTCAUSE_METRICS_SORT_MODE_ASC = 'asc';
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js]: same
```
const ROOTCAUSE_METRICS_SORT_MODE = {
  ASC: 'asc',
  DESC: 'desc'
}
```

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js (hunk 1 above)
Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js]: hmm. the definition yes. the comparison will look worse "ROOTCAUSE_METRICS_SORT_PROPERTY.DATASET". isn't there a better way to do enums in JS?

Comment (justYves) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js]: not with Vanilla JS. But typescript allows for it: 
https://www.typescriptlang.org/docs/handbook/enums.html

**Pull request merged**